src/model/element/interaction.js

```diff
@@ -225,8 +225,8 @@ class Interaction extends Element {
     return _.assign( {}, super.json(), _.pick( this.syncher.get(), _.keys(DEFAULTS) ) );
   }
 
-  toBiopaxTemplate(){
-    return this.association().toBiopaxTemplate();
+  toBiopaxTemplate( getElById ){
```

Comment:
There probably is a typo here. Looks like ``getElById`` must be named as ``transform``. I will fix it.